src/dataset/scads_dataset.py

```diff
@@ -17,17 +17,17 @@ def __init__(self, data_folder: str, division: str, args_path: str):
         self._kgs = read_kgs_from_folder(
             data_folder, division, args.alignment_module, args.ordered
         )
+
         train_links = [(e[0], e[1], 1) for e in self._kgs.train_links]
         valid_links = [(e[0], e[1], 1) for e in self._kgs.valid_links]
         test_links = [(e[0], e[1], 1) for e in self._kgs.test_links]
         super().__init__(
-            self._kgs.kg1,
-            self._kgs.kg2,
-            random.Random(),
-            train_links,
-            valid_links,
-            None,
-            test_links,
+            kg1=self._kgs.kg1,
+            kg2=self._kgs.kg2,
+            rnd=random.Random(),
+            labelled_pairs=train_links + valid_links + test_links,
```

Comment:
I now let the dataset class calculate the split, because it was not correct with the removal of the inner links

Reply:
Make sure the random class gets a fixed seed so the splits are reproducible